### DISCUSSION TITLE
Rofi styles

### DISCUSSION THREAD

**Discussion by dieBakterie** [prasanthrangan/hyprdots#1211] [Q&A] [answered]
Is it normal that the picture in the Rofi styles don't respect the wallpaper anymore? In the past the pictures for the rofi styles where the same as the actual wallpaper.

**Comment by kRHYME7** [comment, marked as answer]
It only happens to me because I'm experimenting something.

``` Hyprdots cache rebuild ``` Should probably fix it (Takes time)

Or I usually do this to fix it. 

First select the problematic  wallpaper using ``` Hyprdots wallpaper select ```
After confirming that the thumbnails are in correct 
run this  
``` find ~/.cache/hyprdots/ -type f -name "*$(basename $(readlink ~/.config/swww/wall.set))*"  -delete ```

then re select the wallpaper again 

https://github.com/prasanthrangan/hyprdots/assets/53417443/a4237527-5abe-4f7b-9f97-cebe2c5fbcd1
